client/usr/local/bin/discover-prometheus-tunnel

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -x
```

Comment:
Can you please replace the existing (unused) script with this one? It's just a rename, really.